projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -809,13 +809,26 @@ function wpcom_launchpad_get_task_definitions() {
 			'is_complete_callback' => 'wpcom_launchpad_is_task_option_completed',
 			'is_visible_callback'  => '__return_true',
 		),
+		// Post-migration tasks.
 		'review_site'                     => array(
 			'get_title'            => function () {
 				return __( "Review the site's content", 'jetpack-mu-wpcom' );
 			},
 			'is_complete_callback' => 'wpcom_launchpad_is_task_option_completed',
 			'is_visible_callback'  => '__return_true',
 		),
+		'connect_migration_domain'        => array(
+			'get_title'            => function () {
+				return __( 'Connect your domain', 'jetpack-mu-wpcom' );
+			},
+			'get_calypso_path'     => function ( $task, $default, $data ) {
+				$domain = get_option( 'migration_source_site_domain', null );
```

Comment:
I'm not convinced that `get_option` will work for fetching the source site domain since I don't know if we'll have access to the pre-transfer site options at this point.

We have a special function to handle this on WP.com but it's specific to WP.com mu-plugins -- is it guaranteed jetpack-mu-wpcom will have access to it? If so, we can switch to that function instead.